maggy/src/providers/__init__.py

```
@@ -0,0 +1,20 @@
+"""Issue tracker provider abstractions."""
+
+from .base import IssueTrackerProvider, Task, Comment
+from .github_issues import GitHubIssuesProvider
+from .asana import AsanaProvider
+
+__all__ = ["IssueTrackerProvider", "Task", "Comment", "GitHubIssuesProvider", "AsanaProvider"]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)